src/scripts/components/component.es6

```diff
@@ -96,7 +96,7 @@ module.exports = class Component {
 	destroy() {
 		this.detachListeners()
 
-		for (let prop in this) {
+		for (const prop in this) {
```

Comment:
No ale co kontextuálně? Jak se ti to čte? Const má znamenat, že se ti nemění reference. Jenže v té smyčce se reference mění.